packages/dds/tree/src/shared-tree/sharedTree.ts

```diff
@@ -603,7 +603,7 @@ export interface SharedTreeOptionsInternal
 }
 /**
  * Configuration options for SharedTree's internal tree storage.
- * @alpha
+ * @beta @input
```

Comment:
These look like unrelated changes. You may need to update your branch.

Reply:
Revert this file